### PR TITLE
[core] Remove dead code for the docs

### PR DESCRIPTION
**Pull request by oliviertassinari**
A follow-up on #4790

https://deploy-preview-4791--material-ui-x.netlify.app/x/advanced-components/

### CODE REVIEW

**Comment by siriwatknp** [comment]
@oliviertassinari Did you use a tool? any way that we can verify?

**Comment by m4theushw** [comment]
The sponsor logos are not very "dead code". Running locally they are used, now we see 404 errors in the console. Should we keep them?

image

**Comment by oliviertassinari** [comment]
@m4theushw Thanks for raising, fixed. @siriwatknp No, I didn't use any tool. Right now, my objective is to remove as many dead files as possible. Maybe a good next step on this problem would be to better organize the `docs/public/static/` folder. We could replicate the URL structure:

- `/ads-in-house/` -> `/docs/ads-in-house/`
- `/blog/` no changes
- `/branding/` no changes
- `/colors-preview/` -> `/material-ui/colors-preview/` and add a 301 for older code
- `/styles/` -> `/docs/styles/`
- `/sponsors/` -> /docs/sponsors/`
- `/images/` -> `material-ui/`

File: docs/public/static/styles/prism-okaidia.css [hunk 1]
```diff
@@ -6,8 +6,14 @@
  * @author ocodia
  */
 
-code[class*="language-"],
-pre[class*="language-"] {
+/* inline code */
+code {
+  padding: 0.1em;
+  border-radius: 0.3em;
```
Comment (oliviertassinari) [inline docs/public/static/styles/prism-okaidia.css:12]: I'm synching with the mono repo.

**Review by siriwatknp — APPROVED** [review]

**Pull request merged**